src/views/Home/components/DaysLeft/index.jsx

```diff
@@ -102,8 +102,8 @@ const DaysLeft = () => {
           name: 'Post-Summer',
           label: 'Fall',
           type: 'Break',
-          start: addDays(termDates.summer?.end, 1),
-          end: addDays(termDates.fall?.start, -1),
+          start: addDays(termDates.fall?.start, -1),
```

Comment:
The new logic here doesn't make sense with the way the other breaks are handled.  Why is the start of post summer defined in terms of the start of the fall semester?  The code you had before seems more correct.

Or - if this logic is in fact correct, does this mean that the logic for winter and preSummer should also be changed?

Reply:
I just noticed the real issue, since the end of the summer and the start of the fall are one day apart, when I add and subtract one from them it essentially switches the two dates. Now I am thinking of either not adding or subtracting for postSummer, but that seems like bugs can appear easily if the dates aren't one day apart. I think the best option would be to add logic that if the days are greater than one day apart then add and subtract accordingly, otherwise don't.

Reply:
I added logic that if the difference between the two dates is greater than 1 it would properly add and subtract the dates, otherwise, it will have them remain as they are and display one day left until fall term when on that day.